admin-ui/plugins/user-management/redux/audit/Resources.ts

```diff
@@ -4,3 +4,4 @@ export const API_PERMISSION = 'api-permission'
 export const API_MAPPING = 'api-role-mapping'
 export const API_USERS = 'api-users'
 export const API_LICENSE = 'api-license'
+export const LDAP = 'api-ldap'
```

Comment:
Rename audit resource constant to avoid collision  

admin-ui/plugins/user-management/redux/audit/Resources.ts: change the export name for clarity.  

```diff
-export const LDAP = 'api-ldap'
+export const API_LDAP = 'api-ldap'
```  

No existing imports of `LDAP` from this file were found, so no call-site updates are currently necessary.